What does this mean?:
```
2019-01-16T23:19:44.0996261Z   Describing Common Tests - Validate Markdown Files
2019-01-16T23:19:44.0996343Z VERBOSE: NPM is checking Gulp is installed. This may take a few moments.
2019-01-16T23:20:08.6540685Z     [+] Should not have errors in any markdown files 25.25s
2019-01-16T23:20:08.6717885Z Tests completed in 159.46s
2019-01-16T23:20:08.6972505Z Tests Passed: 431, Failed: 1, Skipped: 0, Pending: 2, Inconclusive: 0 
2019-01-16T23:20:09.9762947Z markdown link tests failed
2019-01-16T23:20:09.9763679Z At /home/vsts/work/_temp/556706ef-519c-4971-8ccf-5d3722c2cbeb.ps1:8 char:3
2019-01-16T23:20:09.9763748Z +   throw "markdown link tests failed"
2019-01-16T23:20:09.9763840Z +   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2019-01-16T23:20:09.9763892Z + CategoryInfo          : OperationStopped: (markdown link tests failed:String) [], RuntimeException
2019-01-16T23:20:09.9763971Z + FullyQualifiedErrorId : markdown link tests failed
2019-01-16T23:20:09.9764053Z  
2019-01-16T23:20:09.9817530Z ##[error]PowerShell exited with code '1'.
```

--- a/test/common/markdown/markdown.tests.ps1
+++ b/test/common/markdown/markdown.tests.ps1
@@ -54,6 +54,7 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
         try
         {
             $docsToTest = @(
+                './.github/SUPPORT.md'
                 './.github/CONTRIBUTING.md'
                 './*.md'
                 './demos/python/*.md'

